### ISSUE TITLE
Categories collide with text from tags

### ISSUE THREAD

**Issue by maplepy** [aleksilassila/buildshelf#1]
When you go on https://buildshelf.net/builds and click on `All categories` you'll see that the text like `#house, redstone, #medieval` collides with the actual categories, making it hard to read and click.

Also the categories can be scrolled down to get the full list but can't be scrolled back up which makes sense but is weird, maybe either make a maximum size so we can scroll down and up or show the full list from the start?

**Comment by aleksilassila** [comment]
I noticed this too yesterday when I updated all the packages to latest, including radix-ui which is partly responsible for the dropdowns. So far I have no idea what is causing this in the latest version of that library, probably have to go to them for some help on this one.

**Comment by maplepy** [comment]
Is that a javascript or typescript issue? I could try to learn the language to help you ?

**Comment by maplepy** [comment]
Sure, let me know if I can be of any help :)

By the way great website! Love the idea behind it

**Comment by aleksilassila** [comment]
Thanks! The typescript shouldn't affect the final product, only makes the development easier. I would appreciate the help, just be aware that if you don't have previous experience with web stuff, it might be hard to get a good understanding without a lot of practice.

**Comment by maplepy** [comment]
Yeah I figured it might take at least a few weeks just to get going with web dev :/ So far I never really looked into web so yeah it will be a first

I hope my "issues" are not too annoying! I really like the website and there's only 2 litematics websites I know and this one is by far better in terms of UI and Design imo and I'd really like it to be as good as the other in terms of UX
